### PR TITLE
mosdepth 0.2.9

### PR DESCRIPTION
**Pull request by cmdcolin**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by cmdcolin** [comment]
Xref https://github.com/brentp/hts-nim-tools/issues/8#issuecomment-617232917 I saw that the current mosdepth formula was not able to function, likely due to htslib formula update from 1.9 to 1.10

File: Formula/mosdepth.rb [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
   homepage "https://github.com/brentp/mosdepth"
-  url "https://github.com/brentp/mosdepth/releases/download/v0.2.3/mosdepth"
-  sha256 "0aaeb283b296e5ee0a2d819b4842b92289145ed17a37210144f0e02edd3629c4"
+  url "https://github.com/brentp/mosdepth/releases/download/v0.2.9/mosdepth"
+  sha256 "a73283fb1a7465601a4d2d738f6f832f2fd84bf9181e0d4d2b91453da385177c"
 
   bottle do
     root_url "https://linuxbrew.bintray.com/bottles-bio"
     cellar :any_skip_relocation
-    sha256 "bb05121fdf75c04bc21a7ea46008eb3309a4d9f52ccbba0bde202605a77f3f28" => :x86_64_linux
```
Comment (sjackman) [inline Formula/mosdepth.rb:10]: No need to remove the bottle.

File: Formula/mosdepth.rb [hunk 2]
```diff
@@ -20,12 +19,6 @@ class Mosdepth < Formula
 
   def install
     bin.install "mosdepth"
-    unless OS.mac?
-      system "patchelf",
-        "--set-interpreter", HOMEBREW_PREFIX/"lib/ld.so",
-        "--set-rpath", HOMEBREW_PREFIX/"lib",
-        bin/"mosdepth"
-    end
```
Comment (sjackman) [inline Formula/mosdepth.rb:28]: Why did you remove this chunk?

Comment (cmdcolin) [reply Formula/mosdepth.rb:28]: It caused a build error saying it was a static binary, see https://github.com/brewsci/homebrew-bio/runs/605628184

File: Formula/mosdepth.rb (hunk 1 above)
Comment (cmdcolin) [reply Formula/mosdepth.rb:10]: Oh, I dunno if I did that, I used brew bump-formula-pr with --url and --share256 args. It looks like that did it. I can restore?

Comment (sjackman) [reply Formula/mosdepth.rb:10]: Weird. Yes, please. You can make the changes in your editor `brew edit mosdepth` and use `git commit --amend` to modify the commit and `git push --force cmdcolin mosdepth-0.2.9` to update the PR.

File: Formula/mosdepth.rb (hunk 2 above)
Comment (sjackman) [reply Formula/mosdepth.rb:28]: Got it. Thanks!

**Comment by sjackman** [comment]
Merged. Thank you for your first contribution to Brewsci/bio, Colin! 🥇

**Comment by sjackman** [comment]
Due to a new CI workflow, the default development branch is now `develop`. Please run the following commands to update your local working copy.
```
cd $(brew --repo brewsci/bio)
git fetch origin develop
git remote set-head origin -a
```
